presto-product-tests/conf/docker/common/compose-commons.sh

```diff
@@ -15,7 +15,7 @@ function export_canonical_path() {
     elif [[ -f $PATH ]]; then
         PATH=$(echo "$(cd ${PATH%/*} && pwd)/${PATH##*/}")
     else
-        echo "Invalid path: [$PATH]. Working directory is [$(pwd)]."
+        (>&2 echo "Invalid path: [$PATH]. Working directory is [$(pwd)].")
```

Comment:
1. instead of `(>&2` please simply use `>&2` at the end of the echo command
2. cmt msg "Display error on stderr"

`s/compose-compose.sh/docker-compose.sh/`
`s/avoid/to avoid/`
`s/operator//`